src/components/NewDraftWizard/WizardSteps/WizardAddObjectStep.tsx

```diff
@@ -47,14 +50,14 @@ const WizardAddObjectStep: React.FC = () => {
 
   return (
     <>
-      <WizardHeader headerText="Create submission" />
-      <WizardStepper />
+      {/* <WizardHeader headerText="Create submission" />
```

Comment:
do we need to leave these commented out ?

Reply:
Unnecessary commented lines should be gone now.